packages/hw-app-btc/src/Btc.js

```diff
@@ -497,6 +515,19 @@ btc.createPaymentTransactionNew(
       ? this.getTrustedInputBIP143.bind(this)
       : this.getTrustedInput.bind(this);
     const outputScript = Buffer.from(outputScriptHex, "hex");
+    let timestamp = Buffer.alloc(0);
+    if (hasTimestamp) {
+      hasTimestamp().then(
+        date =>
+          (timestamp = Buffer.from(
+            this.toHexInt(date / 1000)
+              .match(/([0-9a-f]{2})/g)
+              .reverse()
+              .join(""),
+            "hex"
+          ))
```

Comment:
I think it's better if we can directly create the Buffer with the number. without using a temporary string conversion